catboost/libs/metrics/metric.cpp

```diff
@@ -614,6 +614,115 @@ void TRMSEMetric::GetBestValue(EMetricBestValue* valueType, float*) const {
     *valueType = EMetricBestValue::Min;
 }
 
+/* Cox partial loss */
```

Comment:
Why partial while it is not? )

Reply:
Because we consider only those examples that have a positive target.
https://en.wikipedia.org/wiki/Proportional_hazards_model